src/frontend/api.ts

```diff
@@ -143,9 +152,113 @@ export const SERVER_START_TIMEOUT = 30000;
 
 const pixelRatio = PixelRatio.get();
 
-interface ApiParam {
-  baseUrl: string;
-  timeout?: number;
+function createRequestClient({
+  baseUrl,
+  onReady,
+}: {
+  baseUrl?: string;
+  onReady?: () => Promise<void>;
+} = {}) {
+  const req = ky.extend({
+    prefixUrl: baseUrl,
+    timeout: false,
+    headers: {
+      "cache-control": "no-cache",
+      pragma: "no-cache",
+    },
```

Comment:
the app server uses these headers so made an assumption that we'd want the same with the map server. not entirely sure if that's a good assumption though

Reply:
Maybe want to re-visit this in a further iteration (beyond this PR). Caching can help reduce CPU use, but causes issues when users update map styles etc.